stories/va-radio.stories.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  getWebComponentDocs,
+  componentStructure,
+  propStructure,
+  StoryDocs,
+} from './wc-helpers';
+
+const radioDocs = getWebComponentDocs('va-radio');
+const radioItem = getWebComponentDocs('va-radio-option');
+
+export default {
+  title: 'Components/va-radio',
+  subcomponents: componentStructure(radioItem),
+  parameters: {
+    docs: {
+      /* eslint-disable-next-line react/display-name */
+      page: () => <StoryDocs docs={radioDocs.docs} />,
+    },
+  },
+};
+
+const Template = args => {
+  return (
+    <va-radio {...args}>
+      <va-radio-option
+        label="Option one"
+        name="one"
+        value="1"
+      ></va-radio-option>
```

Comment:
For simplicity / better readability, I'd recommend making this a self-terminating node.
```suggestion
      <va-radio-option
        label="Option one"
        name="one"
        value="1" />
```